backend/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "example-app"
+version = "0.1.0"
+description = "Example dataspace application backend"
+authors = ["IOXIO"]
+packages = [{include = "app", from = "."}]
+
```

Comment:
imo this should also include the license https://python-poetry.org/docs/pyproject/#license